src/services/wing.services.ts

```diff
@@ -67,24 +67,18 @@ const addCordinator = async (id: string, data: { id: ObjectId }) => {
   }
 };
 
-const updateCordinators = async (
-  id: string,
-  data: { oldId: string; newId: string },
-) => {
+const deleteCoordinator = async (wingId: string, data: { userId: string }) => {
   try {
-    const wing = await wingsRepository.getWingById(id);
+    const wing = await wingsRepository.getWingById(wingId);
     if (!wing) {
       throw new Error("Wing not found");
     }
-    const coordinators = [...wing.coordinators];
-    const index = coordinators.findIndex(
-      (coordinatorId) => coordinatorId.toString() === data.oldId,
+    let coordinators = [...wing.coordinators];
+    coordinators = coordinators.filter(
```

Comment:
@irfan-iiitr Instead of using filter we can directly use mongoose pop method of array to pop specific id from the coordinators array. 
I think pop method is quite easy and straightforward to use here : )